pkg/common/terraform/terraform.go

```diff
@@ -0,0 +1,114 @@
+package terraform
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/hc-install/product"
+	"github.com/hashicorp/hc-install/releases"
+	"github.com/hashicorp/terraform-exec/tfexec"
+)
+
+type runner struct {
+	Context    context.Context
+	Installer  *releases.LatestVersion
+	Runner     *tfexec.Terraform
+	WorkingDir string
+}
+
+// New creates a terraform runner for the provided working directory where
+// terraform files are. It returns a TerraformRunner struct which has the
+// runner to perform any commands terraform-exec package provides.
+func New(workingDir string) (*runner, error) {
+	installer := releases.LatestVersion{
+		Product:    product.Terraform,
+		InstallDir: "/tmp",
```

Comment:
We need to make sure this is writable wherever this runs (pods?)

Reply:
@jbpratt Opened a new PR to introduce the terraform package (https://github.com/openshift/osde2e/pull/1606).

I did a simple test and was able to verify from a pod running in OpenShift, the container is able to write to `/tmp`. We should not run into any issues.